plugins/mac-avcapture/OBSAVCapture.m

```diff
@@ -649,16 +618,14 @@ - (BOOL)updateSessionwithError:(NSError *__autoreleasing *)error
 
 #pragma mark - OBS Settings Helpers
 
-+ (CMVideoDimensions)dimensionsFromSettings:(void *)settings
++ (CMVideoDimensions)legacyDimensionsFromSettings:(void *)settings
```

Comment:
I don't think this needs to be prefixed with "legacy" as it's a getter function to convert the string from settings into an actual `CMVideoDimensions` object. It might only be _used_ by legacy code, but it's still a correct getter method.

Reply:
Hmm, I had it marked as `legacy` because we won't even store the resolution directly as its own key/value any longer, since that value is now encapsulated in the stored format. So it seems like an incorrect getter if it was called `dimensionsFromSettings` unless it also tried to parse the dimensions from the new `supported_format` key, no? Since it won't work unless you're calling it for a source that has the old set of properties.

Reply:
Ah forgive me, you're right of course - the setting itself is not some global, universal, OBS dimension setting, but a setting specifically used for legacy configurations. Ignore my comment then.